plutus-conformance/test-cases/uplc/evaluation/untyped-plutus-core-tests/README.md

```diff
@@ -0,0 +1,2 @@
+The test cases here are all extracted from the tests in
```

Comment:
I'm not sure it's interesting to future people that these tests came from there. I think it would be more helpful to try and categorize them like the other tests.

Reply:
I think it's fine here in the README about our tests if we don't plan to move those. But I think the comments about the origins of the Hachi tests *in the tests* are more concerning because 
(1) I didn't track where the RV tests are coming from like that so it's inconsistent.
(2) The tests may be moved so the comments are not necessarily accurate. In fact RV has re-org their tests since I added them.
(3) The tests are kinda generic so it's not very important where they came from.